src/main/java/org/terasology/sensors/ActivateSensorEvent.java

```diff
@@ -2,6 +2,9 @@
 
 import org.terasology.entitySystem.event.Event;
 
+/**
+ * An event which, when received, will allow the target entity's sensors to detect other entities.
```

Comment:
What you describe here is the (current) default behavior when the event reaches the system currently in place.
That means that the documentation currently is correct, and it is definitely helpful, but it also means that introducing new/other systems may change the behavior this event provokes without changing the event itself. 

Without that additional information it is hard to describe this event, though.